File: src/main/java/com/bernardo/dragonascend/DragonAscend.java
package com.bernardo.dragonascend;

import com.bernardo.dragonascend.ki.KiPlayerManager;
import com.bernardo.dragonascend.race.RacePassives;
import net.fabricmc.api.ModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class DragonAscend implements ModInitializer {

    @Override
    public void onInitialize() {
        KiPlayerManager.register();
        RacePassives.register();
        LOGGER.info("Dragon Ascend initialized!");
    }

    public static final String MOD_ID = "Dragon Ascend";
    public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);
}

File: src/main/kotlin/com/bernardo/dragonascend/DragonAscend.kt
package com.bernardo.dragonascend

import com.bernardo.dragonascend.ki.KiPlayerManager
import net.fabricmc.api.ModInitializer
import org.slf4j.LoggerFactory

object DragonAscend : ModInitializer {
    const val MOD_ID = "dragon_ascend"
    val LOGGER = LoggerFactory.getLogger(MOD_ID)

    override fun onInitialize() {
        KiPlayerManager.register()
        LOGGER.info("Dragon Ascend initialized!")
    }
}
